benchmarks/models_config/bert_cpu.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Are the configs now yaml files? They used to be json files in `benchmarks` folder? Clarify in the documentation what we expect users to do